lib/util/streams.ts

```diff
@@ -0,0 +1,19 @@
+import { Blob } from 'buffer';
+import { Readable } from 'stream';
+import ReadableStream = NodeJS.ReadableStream;
+
+export async function streamToString(
+  stream: Readable | ReadableStream | Blob
```

Comment:
```suggestion
  stream: NodeJS.ReadableStream | Blob
```

Do not use alias import.
`Readable` is a subclass of `NodeJS.ReadableStream`, so only base type is required

Reply:
Here is gist that isn't full type-correct, but works with aws-sdk client library: https://gist.github.com/zharinov/b21f462c5f9599559c69c422d9676fd5